### DISCUSSION TITLE
Servicelookupeditor selected value doesnt save

### DISCUSSION THREAD

**Discussion by rhemasystems** [serenity-is/Serenity#6058] [Help] [answered]
Please can anyone help me with Servicelookupeditor, after selecting the value and saving that field value is empty when the record is saved

![image](https://user-images.githubusercontent.com/50826372/138776947-76f37109-8094-4a30-86d3-9f1bca3011fc.png)

![image](https://user-images.githubusercontent.com/50826372/138860356-77509616-8863-4a39-885e-904c562473f1.png)

**Comment by VictorTomaili** [comment, marked as answer]
you can set lookup Id and text for the same field.

**Reply by rhemasystems** [reply]
Thank you, but please I dont understand you

**Reply by VictorTomaili** [reply]
![image](https://user-images.githubusercontent.com/2023440/139854408-c1de42dd-b5f9-48c5-bdeb-e3f67550f4e0.png)

These two options should be the same field name, "serialnumber" you can set it as option for the `servicelookupeditor`
